WordPress/Classes/ViewRelated/Me/All Domains/Views/AllDomainsListViewController.swift

```diff
@@ -0,0 +1,149 @@
+import UIKit
+import Combine
+
+final class AllDomainsListViewController: UIViewController {
+
+    // MARK: - Dependencies
+
+    private let viewModel: ViewModel
+
+    // MARK: - Views
+
+    private let tableView = UITableView(frame: .zero, style: .insetGrouped)
+
+    // MARK: - Properties
+
+    private lazy var state: ViewModel.State = viewModel.state
+
+    // MARK: - Observation
+
+    private var cancellable = Set<AnyCancellable>()
+
+    // MARK: - Init
+
+    init(viewModel: ViewModel = .init()) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - View Lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.title = Strings.title
+        WPStyleGuide.configureColors(view: view, tableView: nil)
+        self.setupSubviews()
+        self.observeState()
+        self.viewModel.loadData()
+    }
+
+    private func setupSubviews() {
```

Comment:
[Suggestion] I suggest breaking out the search bar and tableview setup into their own helper functions.

Reply:
Resolved in fdf0c0524489872c4a3d2ec75f22ef76a5e6c7e7